pkg/internal/testutils/testutils.go

```diff
@@ -90,20 +91,23 @@ func CheckErrorContainment(t *testing.T, testInfo, expectedErrorMsg, actualErrMs
 const (
 	// the executable we need from graphviz is "dot"
 	executableNameForGraphviz = output.DOTFormat
-	// graph suffix
-	graphSuffix = "png"
-	dotSign     = "."
+	dotSign                   = "."
 )
 
+var graphsSuffixes = []string{"png", "svg"}
+
 // checks if "graphviz" executable exists, if yes runs a cmd to generates a png graph file from the dot output
-func generateGraphFileIfPossible(dotFilePath string) error {
+func generateGraphFilesIfPossible(dotFilePath string) error {
 	// check if graphviz is installed to continue
 	if _, err := exec.LookPath(executableNameForGraphviz); err == nil {
-		pngFilePath := dotFilePath + dotSign + graphSuffix
-		cmd := exec.Command("dot", dotFilePath, "-T"+graphSuffix, "-o", pngFilePath)
-		if err := cmd.Run(); err != nil {
-			return err
+		for _, graphSuffix := range graphsSuffixes {
+			graphFilePath := dotFilePath + dotSign + graphSuffix
+			cmd := exec.Command("dot", dotFilePath, "-T"+graphSuffix, "-o", graphFilePath) //nolint:gosec // nosec
+			if err := cmd.Run(); err != nil {
+				return err
+			}
 		}
 	}
+	logger.NewDefaultLogger().Warnf("dot executable of graphviz was not found. Output Graphs will not be generated")
```

Comment:
missing an `else` before this warning?

Reply:
true , made small rearrange to avoid inner identations